classes/acrn-sblimage.bbclass

```diff
@@ -0,0 +1,88 @@
+inherit python3native
+DEPENDS += "slimboot-tools-native"
+
+MB_DEPENDENCY    ?= ""
+MB_MC_DEPENDENCY ?= ""
+MB_ACRN_BINARY   ?= "${TMPDIR}/deploy/images/${MACHINE}/acrn.32.out"
+MB_ACRN_CMDLINE  ?= ""
+MB_ACRN_MODULES  ?= "${TOPDIR}/conf/linux.txt;${TMPDIR}/deploy/images/${MACHINE}/${KERNEL_IMAGETYPE}"
+BASE_SBLIMAGE    ?= "sbl_os"
+SBLIMAGE_NAME    ?= "${BASE_SBLIMAGE}"
+PREGENERATED_SIGNING_KEY_SLIMBOOT_KEY_SHA256 ?= "${TOPDIR}/cert/TestSigningPrivateKey.pem"
+
+python do_acrn_sblimage() {
+    import re
+    import os
+
+    mbAcrnBinaryDeployDir = d.getVar('MB_ACRN_BINARY').lstrip().rstrip()
+    if not os.path.isfile(mbAcrnBinaryDeployDir):
+        bb.error("acrn %s not found!" % mbAcrnBinaryDeployDir)
```

Comment:
Would it help if we can give some helpful hints to the user when this happens? When can this happen?

Reply:
Also, do we expect the task to fail here? bb.error doesn't cause the task to fail I think. Perhaps raise an exception or check if bb.fatal make more sense.

Reply:
the task should fail. I will use bb.fatal. In normal case this never happens. Because if ACRN hypervisor build failed, bitbake will not reach at here. It could happen only when user moved acrn.32.out file like below. 

$ bitbake mc:sos:acrn-hypervisor
$ rm master-acrn-sos/deploy/images/intel-corei7-64/acrn.32.out 
$ bitbake mc:sos:acrn-image-minimal

In this case bitbake will run the do_acrn_sblimage() task because it is not aware of that acrn.32.out is missing.

Reply:
the error message will show file path which is not available. for example,

acrn master-acrn-sos/deploy/images/intel-corei7-64/acrn.32.out not found!

It should be enough hint for users to understand the failure reason.